pbs_light.go

```diff
@@ -618,16 +618,58 @@ func init() {
 	viper.AddConfigPath(".")
 	viper.AddConfigPath("/etc/config")
 
+	// Some defaults will be set just so they are accessable via environment variables
+	// (basically so viper knows they exist)
```

Comment:
Could you start this comment with `// Fixes #475: {description}`?

It's searchable by grep/find/whatever, and points confused engineers to the discussion which caused the code.